From 1843bb59f94775eaec2162e661bdf8c891b257e2 Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Tue, 15 May 2018 14:44:55 +0300
Subject: [PATCH 0620/1051] dts: a3700: fix eip97 node

The node is missing cell-index tag which the driver needs.

Change-Id: I9fb9cf629764a5354276df927b81e8a0a2f0f875
Signed-off-by: Ofer Heifetz <oferh@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index d9531e242eb4..e52dfa889bd3 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -319,6 +319,7 @@
 				interrupt-names = "mem", "ring0", "ring1",
 						  "ring2", "ring3", "eip";
 				clocks = <&nb_periph_clk 15>;
+				cell-index = <0>;
 			};
 
 			sdhci1: sdhci@d0000 {
-- 
2.17.1

